src/br/unicamp/meca/mind/MecaMind.java

```diff
@@ -277,140 +293,189 @@ private void mountMotivationalCodelets() {
 		}
 	}
 
-	private void mountMotivationalBehavioralCodelets() {
-		if (getMotivationalBehavioralCodelets() != null) {
-			for (MotivationalBehavioralCodelet motivationalBehavioralCodelet : getMotivationalBehavioralCodelets()) {
-				if (motivationalBehavioralCodelet != null && motivationalBehavioralCodelet.getId() != null
-						&& motivationalBehavioralCodelet.getMotivationalCodeletsIds() != null
-						&& motivationalBehavioralCodelet.getMotorCodeletId() != null) {
+	private void mountBehaviorCodelets() {
+		if (behaviorCodelets != null) {
+			for (BehaviorCodelet behaviorCodelet : behaviorCodelets) {
```

Comment:
Seria interessante utilizar conceitos do Java 8 com lambda expression. Neste caso
```suggestion
behaviorCodelets.forEach(behaviorCodelet -> 
```

Reply:
Nice suggestion, indeed, but as is just a matter of style, resulting in the same end, will let it to a future PR.